.github/workflows/build.yml

```diff
@@ -70,9 +79,26 @@ jobs:
 
     - name: Compile library
       run: make -j
+      if: ${{ !matrix.useCMake }}
```

Comment:
Please add this line also to the `Compiler info` step. Printing the make settings for a CMake build doesn't make much sense.